src/types/select.js

```diff
@@ -4,12 +4,42 @@ import assign from "lodash.assign";
 import css from "./select.css";
 import multiple from "./lib/multiple.js";
 
+function findSelected(children) {
+    return children.find((option) => Boolean(option.selected));
```

Comment:
`option.selected` should already be a boolean, so I don't think the cast is necessary in this instance.

I don't hate `!!`, I think it's a neat trick that obscures intent and is easy to gloss over while reading. Being clear is more important than being concise.